DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionCardViewModelTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Data.Models.CourseContent;
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.LearningMenu;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    class SectionCardViewModelTest
+    {
+        [Test]
+        public void Section_should_return_empty_string_if_has_learning_is_false()
+        {
+            // Given
+            const string sectionName = "TestName";
+            const bool hasLearning = false;
+            const double percentComplete = 12.00;
+            var section = new CourseSection(sectionName, hasLearning, percentComplete);
```

Comment:
Probably worth creating a helper method like we have for creating course content, so that you can only override the fields you care about when constructing a section 🙂